src/components/question/Question.jsx

```diff
@@ -0,0 +1,24 @@
+import "./css/questionStyle.css";
+
+export default function Question({
+  currentQuestion,
+  optionClickHandler,
+  hasUserSelected,
+}) {
+  return (
+    <div className="quiz-question-container --bold-600 --has-padding">
+      <p className="question">{currentQuestion.question}</p>
+      {Object.keys(currentQuestion.options).map((option, index) => (
+        <button
+          className="option --verticle-flex --centered-flex"
+          value={currentQuestion.options[option]}
+          key={`${currentQuestion.questionCode}-${index}`}
+          onClick={(e) => optionClickHandler(e)}
+          disabled={hasUserSelected ? true : false}
```

Comment:
actual you don't need to write true false explicitly. "hasUserSelected" is already a boolean value.